.github/workflows/e2e.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "End to End Tests"
+on:
+  push:
+    # run on pushes to master or release/*
+    branches:
+      - master
+      - release/*
+  pull_request:
+    # run on pull requests targeting master
+    branches:
+      - master
+jobs:
+  end-to-end:
+    runs-on: [self-hosted, e2e-worker]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Clean Project State
+      run: |
+        # clean GCP resources
+        .github/workflows/e2e_scripts/clean.sh
+    - name: Build Sandbox Services
+      run: |
+        set -x
+        skaffold config set --global local-cluster false
+        # tag with git hash
+        skaffold build --default-repo=gcr.io/$PROJECT_ID \
+                       --tag=$GITHUB_SHA
+      env:
+        PROJECT_ID: ${{ secrets.E2E_PROJECT_ID  }}
+    - name: Rewrite Manifests
+      run: |
+        set -x
+        OLD_PATH=gcr.io/stackdriver-sandbox-230822
+        NEW_PATH=gcr.io/stackdriver-sandbox-e2e/$OLD_PATH
```

Comment:
please indicate "Tests" in the name

Reply:
`gcr.io/stackdriver-sandbox-e2e/tests/gcr.io/stackdriver-sandbox-230822`?

Note that this is in a different project than the canonical images, so we don't have to worry about them getting mixed up

Reply:
yeah, this name works

Reply:
done